manifests/controllers/srlinux/manifest.yaml

```
@@ -394,7 +394,7 @@ spec:
         - --leader-elect
         command:
         - /manager
-        image: ghcr.io/srl-labs/srl-controller:0.3.1
+        image: ghcr.io/srl-labs/srl-controller:0.4.4
```

```suggestion
        image: ghcr.io/srl-labs/srl-controller:0.4.5
```
in 0.4.5 the version parsing function has been fixed to allow tags such as `version_X.Y.Z-buildnum`